docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2115,11 +2115,11 @@ The following error message codes may appear on the app-server log. Use the foll
 
 ### ZWED0086W
 
-  Could not stop tomcat, error=_errorMessage_
+  Could not stop Tomcat, error=_errorMessage_
 
   **Reason:**
 
-  If Tomcat manager unable to stop tomcat-process for Unix then it logs _errorMessage_.
+  If Tomcat manager is unable to stop the Tomcat process on Unix, then it logs _errorMessage_.
 
   **Action:**
 
```

Comment:
Review the _errorMessage_ and if app server service is interrupted, send it to support for troubleshooting.